jenkins-build-k8s-snaps.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+KUBE_VERSION=${KUBE_VERSION:-"v1.5.2"}
```

Comment:
Only because I saw it elsewhere: https://github.com/juju-solutions/kubernetes-jenkins/blob/f1587326589b48d6d1370972da0f57335638e21d/build-kubernetes.sh#L15